<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDockWidget</class>
 <widget class="QDockWidget" name="ProjectDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>491</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>186</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pro&amp;jects</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxProjects"/>
    </item>
    <item>
     <widget class="QLineEdit" name="filterLineEdit">
      <property name="placeholderText">
       <string>filter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeViewProject">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
